libs/portals/core/src/lib/messages.ts

```diff
@@ -54,6 +54,19 @@ export const m = defineMessages({
     defaultMessage: 'Tókst ekki að sækja',
     id: 'portals:could-not-fetch',
   },
+  dashboardIntro: {
+    defaultMessage:
+      'Hér eru stjórnborð sem þú hefur aðgang að fyrir ýmsar þjónustur tengdar Ísland.is.',
+    id: 'portals:dashboard-title',
+  },
+  dashboardTitle: {
+    defaultMessage: 'Stjórnborð Ísland.is',
+    id: 'portals:dashboard-title',
+  },
+  dashboardImageAlt: {
+    defaultMessage: 'Mynd af byggingu sem táknar stjórnborð',
+    id: 'portals:dashboard-img-alt',
+  },
```

Comment:
Don't we have an admin portal specific translation namespace?

Reply:
Ahh yeah good point. We have not created yet `libs/portals/admin/core` library. I will add it and have the translations as the first thing defined there